elixir.md

```diff
@@ -145,6 +150,29 @@ elixir(function(mix) {
 });
 ```
 
+While it's recommended that you use Laravel's default asset directories, if you require a different base directory, you may begin any file path with `./`. This instructs Elixir to begin at the project root, rather than using the default base directory.
+
+```javascript
+elixir(function(mix) {
+    mix.sass('./app/assets/app.scss');
+});
+```
+
+With this adjustment, Elixir will now compile `./app/assets/sass/app.scss` to `public/css/app.css`.
```

Comment:
Should this read `./app/assets/app.scss`? Or should line 157 read `./app/assets/sass/app.scss`?